examples/pxScene2d/src/pxHttpCache.cpp

```diff
@@ -1,3 +1,4 @@
+#ifdef ENABLE_HTTP_CACHE
```

Comment:
Let's remove this #if condition.  I think it's best if we compile everything by default.  The ENABLE_HTTP_CACHE should just enable this capability, but it should be compiled by default (to ensure others don't break the API contract accidentally).